ENIGMAsystem/SHELL/Universal_System/image_formats.h

```diff
@@ -35,6 +35,12 @@ struct RawImage {
   RawImage(RawImage &&other): pxdata(other.pxdata), w(other.w), h(other.h) {
     other.pxdata = nullptr;
   }
+  void resize(unsigned w, unsigned h) {
+    delete[] pxdata;
+    pxdata = new unsigned char[w*h];
```

Comment:
shouldn't we be assuming brgra so * 4 here? or you could add a channels field to rawimage if you want. Also im not really sure this is a "resize" as you're just cobtructing a new blank img.

Reply:
yeah d579896848740734428c47543bb793966dc68c90